pkg/kubernetes/kubernetes.go

```diff
@@ -12,12 +12,13 @@ import (
 	"time"
 
 	"github.com/dapr/cli/pkg/print"
+	"github.com/dapr/cli/pkg/version"
 	"github.com/dapr/cli/utils"
 
 	"github.com/briandowns/spinner"
 )
 
-const daprManifestPath = "https://daprreleases.blob.core.windows.net/manifest/dapr-operator.yaml"
+var daprManifestPath string = "https://github.com/dapr/dapr/releases/download/v" + version.GetCLIVersion() + "/dapr-operator.yaml"
```

Comment:
Try to call `RootCmd.Version` from here instead of `version.GetCLIVersion()`.

Reply:
Even that's causing cyclic dependency. We import dapr/cli/pkg/kubernetes package in dapr/cli/cmd (init.go) so we can't really import dapr/cli/cmd in dapr/cli/pkg/kubernetes package. This is based on my understanding so far but correct me if I am wrong.

Reply:
Best way is to give the version as a parameter to Init(), and then put it in the URL. Then you can use RootCmd.Version from the init command.

Reply:
Actually we are not required to match the CLI version, we should get the latest version as suggested by @youngbupark below.